src/app/settings/action-triggers/components/edit-action-triggers/edit-action-triggers.component.ts

```diff
@@ -0,0 +1,150 @@
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChange, SimpleChanges } from '@angular/core'
+import { AbstractControl, FormBuilder, UntypedFormGroup, Validators } from '@angular/forms'
+import { ActionTrigger, CreateActionTrigger, KeyboardAndSelectionTriggerData, KeyboardTriggerData, SHORTCUT_KEYS_MAPPINGS } from 'src/app/shared/models/action-trigger'
+import { Tv2PartAction } from 'src/app/shared/models/tv2-action'
+import { ActionStateService } from 'src/app/shared/services/action-state.service'
+import { Logger } from 'src/app/core/abstractions/logger.service'
+import { ActionTriggerService } from 'src/app/shared/abstractions/action-trigger.service'
+
+export interface ActionTriggerUIForm {
```

Comment:
I remove that interface because I already work with string[] in to the form witch is almost same is Keys. We don't need it to be a string. Now we add Pipe to fix UI logic display.
Fixed